Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -454,6 +474,13 @@ int ReadFromHandle(HANDLE& dev_handle, u8* buf)
   return read;
 }
 
+bool IsValidVendorIDAndProductID(const u16 vendor_id, const u16 product_id)
+{
+  if (vendor_id != 0x057e)
```

Comment:
I was checking BlueZ source code for an unrelated reason, and noticed [they also list a second vendor ID](https://github.com/bluez/bluez/blob/master/plugins/wiimote.c#L50), supposedly for the LEGO Wiimote. Should we add that VID to Dolphin too?

Reply:
On the other hand, looking around it seems like may also have been used for the [PS3 Blue-ray controller](https://forum.kodi.tv/printthread.php?tid=50717&page=43)?

It'd be good to get more testing on that before making that change, and in any case I'd rather do that separately to make pinning down any potential regressions easier.